components/src/dynamo/common/tests/configuration/test_kv_router_args.py

```
@@ -402,27 +402,40 @@ def test_load_aware_preserves_cache_hit_weights() -> None:
     assert kwargs["disk_cache_hit_weight"] == 0.1
 
 
-def test_kv_router_kwargs_preserves_explicit_queue_tiers() -> None:
+def test_kv_router_kwargs_preserves_policy_config_path() -> None:
     parser = argparse.ArgumentParser()
     KvRouterArgGroup().add_arguments(parser)
 
     config = KvRouterConfigBase.from_cli_args(parser.parse_args([]))
-    config.router_queue_by_incoming_missing_isl = [(0, 2048), (2048, 512)]
+    config.router_policy_config = "/tmp/router-policy.yaml"
```

Ruff (0.15.17)

[error] Probable insecure usage of temporary file or directory: "/tmp/router-policy.yaml"

(S108)